CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1,4 @@
+Replace jshint with eslint
+Add prettier code formatting
+Add husky and lint-staged
```

Comment:
What about?

```suggestion
Update codebase to use ES6
-  Remove JSHint and jshint overrides
-  Add esLint using standard tamia presets
-  Replace var with let/const
-  Fix or disable eslint errors
Add prettier code formatting
Add husky and lint-staged
```

To make it closer to the one used in other similar PRs, e.g. https://github.com/telefonicaid/iotagent-json/pull/456/files#diff-0bc49fac4e713c5ac9a5c346f2286f5eef1a765788273dcf4ad32b5711033e6b

Reply:
Fixed in 68cebf8